### DISCUSSION TITLE
Unable to upload an image while applying to GitHub Global Campus

### DISCUSSION THREAD

**Discussion by Nwabunike** [community/community#37964] [GitHub Education]
The option for uploading an image suddenly disappeared while applying for the Github Global Campus. I have cleared my cache and tried different browsers but the issue persists.
I need help, please!

![IMG_8884](https://user-images.githubusercontent.com/109210880/199213567-37116581-1691-4af6-9bfa-eafefa63faa1.png)

**Comment by Addax101** [comment]
well, have you considered having all the document borders well positioned?

**Reply by Nwabunike** [reply]
The issue is there is no button to upload any document. Only the option to use camera as shown above.

**Comment by ossanna16** [comment]
Hi there @Nwabunike and welcome to our community! Thank you for reporting this 🙂
